src/main/java/uk/gov/hmcts/dts/fact/entity/Facility.java

```diff
@@ -1,29 +1,39 @@
 package uk.gov.hmcts.dts.fact.entity;
 
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.JoinTable;
-import javax.persistence.ManyToOne;
-import javax.persistence.Table;
+import java.util.List;
+import javax.persistence.*;
+
 
 @Entity
 @Table(name = "search_facility")
 @Getter
 @Setter
+@NoArgsConstructor
 public class Facility {
 
+    @ManyToMany(mappedBy = "facilities")
+    private List<Court> courts;
+
     private static final String FACILITY_ID = "facility_id";
 
     @Id
+    @SequenceGenerator(name = "seq-gen", sequenceName = "search_facility_id_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq-gen")
     private Integer id;
     private String description;
     private String descriptionCy;
     private String name;
+    @Column(name = "name_cy")
     private String nameCy;
+    private String image;
```

Comment:
Are these 3 fields for image required to be returned to the admin portal? I can see similar one for the facility type. If they are not used anywhere, don't even need to be defined in the entity class

Reply:
they are set to not null field in the search_facility table so need them to update records